app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -451,7 +498,9 @@ def save(self, *args, **kwargs):
 
 
 class NimbusReadyForReviewSerializer(
-    NimbusStatusRestrictionMixin, serializers.ModelSerializer
+    StatusValidationMixin,
+    PublishStatusValidationMixin,
```

Comment:
Wait isn't this serializer read only?  Why does it need the status validators?

Reply:
Hmm, yeah I think these mixins in NimbusExperimentSerializer should cover it. This serializer should just be for final data validation, the other should catch attempts to update at the wrong points